dockerfiles/nginx/proxito.conf

```diff
@@ -62,6 +62,8 @@ server {
         add_header X-RTD-Project-Method $rtd_project_method always;
         set $rtd_redirect $upstream_http_x_rtd_redirect;
         add_header X-RTD-Redirect $rtd_redirect always;
+        set $cdn_cache_control $upstream_http_cdn_cache_control;
+        add_header CDN-Cache-Control $cdn_cache_control always;
```

Comment:
What happens if proxito does not return this header? Are we returning the header anyways with a blank value?